api/v1beta2/cluster_types.go

```diff
@@ -585,6 +585,15 @@ type Topology struct {
 	// +listMapKey=name
 	// +kubebuilder:validation:MaxItems=1000
 	Variables []ClusterVariable `json:"variables,omitempty"`
+
+	// nodeDeletionStrategy specifies the strategy to delete nodes in the cluster.
+	// Available options:
+	// - graceful
+	// - force
+	// By default it's "force".
+	// +optional
+	// +kubebuilder:validation:Enum=force;graceful
```

Comment:
Would be good to explain in the godoc what the two options actually mean. That way someone could `kubectl explain` and know exactly what the two options will do for them